static/js/controllers/contacts.js

```diff
@@ -290,8 +293,30 @@ var _ = require('underscore'),
         $scope.setLeftActionBar(data);
       };
 
-      var setupPromise = getUserHomePlaceSummary().then(function(home) {
-        usersHomePlace = home;
+      var setupPromise = $q.all([
```

Comment:
I wanted to to use the object style all so you could refer to results by key, but that breaks out unit test `Q` implementation.